src/SFA.DAS.RoATPService.Application/Handlers/UpdateOrganisationHandler.cs

```diff
@@ -35,6 +37,10 @@ public class UpdateOrganisationHandler : IRequestHandler<UpdateOrganisationReque
 
         public async Task<bool> Handle(UpdateOrganisationRequest request, CancellationToken cancellationToken)
```

Comment:
I think this handler is obsolete now, but a good idea to cover off all endpoints